### ISSUE TITLE
TO-DO: Chargen Master List

### ISSUE THREAD

**Issue by jaborsh** [jaborsh/Arcellia#30]
1. Screenreader does not adequately provide selections. Add descriptors to options to ensure the point is more readily made on what a user should be doing.

**Comment by jaborsh** [comment]
Screenreader support accomplished.
